usr/share/rear/conf/default.conf

```diff
@@ -373,8 +373,11 @@ LANG_RECOVER=C
 FDRUPSTREAM_INSTALL_PATH="/opt/fdrupstream"
 COPY_AS_IS_FDRUPSTREAM=( "$FDRUPSTREAM_INSTALL_PATH" )
 COPY_AS_IS_EXCLUDE_FDRUPSTREAM=( "$FDRUPSTREAM_INSTALL_PATH/usserver" )
-#CHECK_CONFIG_FILES_FDRUPSTREAM=( "$FDRUPSTREAM_INSTALL_PATH/upstream.cfg" "$FDRUPSTREAM_INSTALL_PATH/uscmd1" "$FDRUPSTREAM_INSTALL_PATH/usdaemon2" )
+CHECK_CONFIG_FILES_FDRUPSTREAM=( "$FDRUPSTREAM_INSTALL_PATH/uscmd*" "$FDRUPSTREAM_INSTALL_PATH/usdaemon*" "$FDRUPSTREAM_INSTALL_PATH/us" "$FDRUPSTREAM_INSTALL_PATH/us1" "$FDRUPSTREAM_INSTALL_PATH/us.ser" "$FDRUPSTREAM_INSTALL_PATH/*.so" "$FDRUPSTREAM_INSTALL_PATH/usudb*" "$FDRUPSTREAM_INSTALL_PATH/*.dat" "$FDRUPSTREAM_INSTALL_PATH/*.cfg" )
```

Comment:
I did test with <code>rear checklayout</code>, and the quoted patterns do work.  But I've just now tested without the quotes and they work as well.  Is there reason for concern that future code changes in <code>51_compare_files.sh</code> might make the quoted patterns stop working?  I'm not opposed to removing the quotes if you think they are risky.

Regarding <code>FDRUPSTREAM_INSTALL_PATH</code>, let me give that some more thought.  My plan was that if <code>FDRUPSTREAM_INSTALL_PATH</code> were to be changed in <code>local.conf</code>, the user would need to copy the rest of the <code>\*FDRUPSTREAM*</code> variables from <code>default.conf</code> to <code>local.conf</code> -- but I will look into other ways of handling this (like in the <code>prep</code> stage, as you suggested).

Thanks for the input.